fe/fe-core/src/main/java/org/apache/iceberg/StarRocksIcebergTableScan.java

```diff
@@ -379,8 +419,11 @@ public CloseableIterable<CombinedScanTask> planTasks() {
                 planFiles(), targetSplitSize(), splitLookback(), splitOpenFileCost());
     }
 
-    // For fe ut
-    public PlanMode getPlanMode() {
-        return planMode;
+    private int liveFilesCount(List<ManifestFile> manifests) {
+        return manifests.stream().mapToInt(this::liveFilesCount).sum();
+    }
+
+    private int liveFilesCount(ManifestFile manifest) {
+        return manifest.existingFilesCount() + manifest.addedFilesCount();
```

Comment:
what's this meaning? why need to add?

Reply:
it's used for adapting iceberg metrics reporter. The metrics can be found by `trace values` or query profile.